udata/tests/test_discussions.py

```diff
@@ -37,6 +37,7 @@ def create_app(self):
         return app
 
     def test_new_discussion(self):
+        self.app.config['FORCE_METRICS'] = True
```

Comment:
Agreed :+1: : you should factorize.
Even better, you should write a `MetricsSettings` class for this case. See https://github.com/opendatateam/udata/blob/master/udata/tests/frontend/test_markdown.py#L16-L21 for an example (it's the pattern we use when we need to activate some settings)